scripts/app

```diff
@@ -22,6 +22,14 @@ Commands:
 EOF
 }
 
+check_if_not_already_running() {
+  if ps ax | grep $0 | grep -v $$ | grep bash | grep -v grep
```

Comment:
Is this a reliable check?

Won't it match any process that contains the string `app`?

Maybe it would be better to use a lockfile.

Reply:
Also would be good to stick to the current convention of `if condition; then` all on the same line.

Reply:
> Also would be good to stick to the current convention of if condition; then all on the same line.

I copied this check from karen without changes, you should probably fix it there